### PR TITLE
Add integration tests & fixes

### PR DESCRIPTION
**Pull request by ErikKalkoken**
- Adds first integrations tests for forum UI
- Fix: show "Last modified" only when message was updated
- Fix: sort message by creation timestamp, not updated timestamp, so updated messages stay in position

### CODE REVIEW

File: aa_forum/templates/aa_forum/partials/search/search-form.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {% load i18n %}
 
-<form autocomplete="off" action="{% url 'aa_forum:search_results' %}" method="get">
+<form id="aa-forum-form-search" autocomplete="off" action="{% url 'aa_forum:search_results' %}" method="get">
```
Comment (ppfeufer) [inline aa_forum/templates/aa_forum/partials/search/search-form.html:3]: image

`/forum/search/`

File: aa_forum/templates/aa_forum/partials/forum/board/board-index.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
     </div>
 </div>
 
-{% include "aa_forum/partials/forum/board/new-topic-button.html" %}
+{% include "aa_forum/partials/forum/board/new-topic-button.html" with num=2 %}
```
Comment (ppfeufer) [inline aa_forum/templates/aa_forum/partials/forum/board/board-index.html:34]: Arguments that are added to an ID string should be descriptive.

position=above-topic-list / below-topic-list

**Review by ppfeufer — CHANGES_REQUESTED** [review]

File: aa_forum/templates/aa_forum/partials/forum/board/board-index.html (hunk 2 above)
Comment (ErikKalkoken) [reply aa_forum/templates/aa_forum/partials/forum/board/board-index.html]: fixed

File: aa_forum/templates/aa_forum/partials/search/search-form.html (hunk 1 above)
Comment (ErikKalkoken) [reply aa_forum/templates/aa_forum/partials/search/search-form.html]: fixed

**Review by ppfeufer — APPROVED** [review]

**Comment by ppfeufer** [comment]
Thank you very much, merging.

**Pull request merged**